metadata.yml

```
@@ -5,7 +5,16 @@ author:
   - Jose Storopoli
   - Rik Huijzer
   - Lazaro Alonso
-html-license: '<a href="http://creativecommons.org/licenses/by-nc-sa/4.0/">CC BY-NC-SA 4.0</a>'
+html-license: >-
+  <span style="display: flex; justify-content: center;">
```

```suggestion
 <span style="width: 200px; height: 35px; display: flex; justify-content: center; border: 1.5px solid rgb(67, 72, 54);border-radius: 10px;">
```
we could go for red, green or completely black, now is kinda dark and probably is already ok. Any colour will do, feel free to merge @rikhuijzer, with the suggestion or without it.